presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -83,4 +94,31 @@ public BaseJdbcConfig setJdbcTypesMappedToVarchar(String jdbcTypesMappedToVarcha
         this.jdbcTypesMappedToVarchar = ImmutableSet.copyOf(Splitter.on(",").omitEmptyStrings().trimResults().split(nullToEmpty(jdbcTypesMappedToVarchar)));
         return this;
     }
+
+    @Min(0)
+    @Max(38)
+    public Integer getDecimalDefaultScale()
+    {
+        return decimalDefaultScale;
+    }
+
+    @Config("decimal-default-scale")
+    public BaseJdbcConfig setDecimalDefaultScale(Integer decimalDefaultScale)
+    {
+        this.decimalDefaultScale = decimalDefaultScale;
+        return this;
+    }
+
+    @NotNull
+    public BaseJdbcConfig.DecimalOverMaxPrecisionMapping getDecimalOverMaxPrecisionMapping()
+    {
+        return decimalOverMaxPrecisionMapping;
+    }
+
+    @Config("decimal-over-max-precision-mapping")
```

Comment:
Just handle rounding here for now? decimal.rounding-mode?

I don't like it mixes different kind of things here. Rounding and mapping to varchar. I used to had https://github.com/prestosql/presto/pull/1182 which was addressing, but I thought it got obsolete because of `jdbc-types-mapped-to-varchar`.

Reply:
`STRING` is required for the user to see what is the actual raw data in case that precision is larger than 38 (`Decimal overflow`)